packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1412,6 +1412,49 @@ Scenario that returns a different file encoding depending on the accept header.
 - image/png return a png image
 - image/jpeg return a jpeg image
 
+### Payload_Pageable_list
+
+- Endpoint: `get /payload/pageable`
+
+List users.
+
+Expected query parameter:
+skip=5
+maxpagesize=3
+
+Expected response body:
+
+```json
+{
+  "value": [
+    {
+      "name": "user5"
+    },
+    {
+      "name": "user6"
+    },
+    {
+      "name": "user7"
+    }
+  ],
+  "nextLink": "{endpoint}//payload/pageable?skip=8&maxpagesize=3"
+}
+```
+
+Expected query parameter:
+skip=8
+maxpagesize=3
```

Comment:
SDK may need to be careful not to send `maxpagesize` twice here, as `nextLink` already have `maxpagesize` parameter

Reply:
are languages generating the `maxpagesize` parameter for DPG? I know Python isn't because we want to expose the ability to toggle page size through our paging model. Wondering where other languages stand cc @joheredi @srnagar @m-nash

Reply:
Java and .NET also hide it from parameter. The page mode would handle them (for Java it is `PagedIterable.iteratorByPage(pageSize)`. I think JS is under discussion as well.

That is the reason I write this in doc
```
SDK may hide the "maxpagesize" from API signature. The functionality of "maxpagesize" could be in related language Page model.
```

I use "may" so the case here won't dictate what SDK should behavior, but at least give a hint here.